test/uk/gov/hmrc/ngrloginregisterfrontend/utils/NGRLoggerSpec.scala

```diff
@@ -17,10 +17,19 @@
 package uk.gov.hmrc.ngrloginregisterfrontend.utils
 
 import uk.gov.hmrc.ngrloginregisterfrontend.helpers.TestSupport
+import uk.gov.hmrc.ngrloginregisterfrontend.util.NGRLogger
 
 class NGRLoggerSpec extends TestSupport {
-
+  val logger: NGRLogger = inject[NGRLogger]
+  // todo: Figure out how to check logs
```

Comment:
If you put to do in capitals e.g. TODO then it will be recognised by intelliJ as something that needs to be done.